roms_tools/setup/cdr_forcing.py

```diff
@@ -698,82 +573,25 @@ def plot_location_side_view(self, release: str = None):
         ValueError
 
             If `self.grid` is not set.
-            If the specified `release` does not exist in `self.releases`.
-            If no `release` is provided when multiple releases are available.
+            If the specified `release_name` does not exist in `self.releases`.
+            If no `release_name` is provided when multiple releases are available.
         """
         if self.grid is None:
             raise ValueError(
                 "A grid must be provided for plotting. Please pass a valid `Grid` object."
             )
 
-        valid_releases = [r for r in self.releases if r != "_tracer_metadata"]
-        if release is None:
-            if len(valid_releases) == 1:
-                release = valid_releases[0]
+        valid_release_names = [r.name for r in self.releases]
+
+        if release_name is None:
```

Comment:
consider reducing nesting:

```py
if release_name is None and len(valid_release_names) != 1:
  raise ValueError(...)

release_name = valid_release_names[0]
```

Reply:
This won't work because it always overwrites `release_name`, even when the user has already specified it (i.e., it's not None).